module3/preparation/Week3/ReadingRelatedResources.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Reading Related Resources Preparation
+---
+
+## Pre-Work for [Reading Related Resources](/module3/lessons/Week3/ReadingRelatedResources)
+
+In our [RESTful Routes](/module3/lessons/Week2/RestfulRoutes.md) lesson, we learned that in order to indicate which resource we are trying to include on a specic show page.  As in `/movies/12` which will return the show page for the movie with an id of 12.
+
+Now, we need to expand our understanding of routing to access _related_ resources. 
```

Comment:
This pre-work looks good to me! I'm liking the plan of making it super clear what the deliverable is.